### PR TITLE
Personalized/municipality-specific groups

### PR DESCRIPTION
**Pull request by Valioesi**
Resolves: https://expedition-grundeinkommen.monday.com/boards/853311751/pulses/2017185988

New component (can be controlled by Contentful) to show groups (Telegram, Facebook, etc). 
The component behaves differently depending on state of app.
1) On municipality page: shows groups for this municipality (same if authenticated or not), if there are no groups yet a small info text is rendered. 
2) On any other page: 
    2.1) Authenticated: Listing groups for all municipalities user is singed up for
    2.2) Not Authenticated: Short info text and sign up button

I tried to design this as flexible as possible since I wasn't shure in the beginning what I should be aiming for. 
I already created sections in Contentful on homepage and municipality page. Need to be deactivated before merging, if we don't want to show the sections yet.
I only added groups for Berlin to the database, but I am going to transfer all of the groups we already have during the course of the day (thursday, 16th).

### CODE REVIEW

**Review by schaerta — APPROVED** [review]
Great, thanks!

File: src/context/Municipality/index.js [hunk 1]
```diff
@@ -64,6 +71,8 @@ export const MunicipalityProvider = ({ children }) => {
     getAllMunicipalityStats();
   }, []);
 
+  console.log('municipality', municipality);
```
Comment (nmauersberg) [inline src/context/Municipality/index.js:74]: Maybe we could remove these logs.

File: src/context/Municipality/index.js [hunk 2]
```diff
@@ -148,6 +159,13 @@ export const MunicipalityProvider = ({ children }) => {
     setMunicipalitiesInObject(municipalityObject);
   }, [allMunicipalityStats]);
 
+  useEffect(() => {
+    console.log({ singleMunicipalityData });
```
Comment (nmauersberg) [inline src/context/Municipality/index.js:163]: Same here

**Review by nmauersberg — APPROVED** [review]
Very nice, thank you. Looks good!

File: src/context/Municipality/index.js (hunk 1 above)
Comment (Valioesi) [reply src/context/Municipality/index.js:74]: Classic Vali :D

**Pull request merged**